internal/controller/operator/factory/vmagent/rbac.go

```diff
@@ -229,11 +231,13 @@ func buildVMAgentClusterRoleBinding(cr *vmv1beta1.VMAgent) *rbacv1.ClusterRoleBi
 func buildVMAgentClusterRole(cr *vmv1beta1.VMAgent) *rbacv1.ClusterRole {
```

Comment:
It'd be great to have migration path for this case. Or add docs for that.

Because after this change, `clusterRoleBinding` and `clusterRole` with previous name will be forever orphaned at kubernetes cluster. 

 Moreover, previous `clusterrole` will still point to the `serviceAccount` and it may confuse users.

Reply:
okay, update note might not be good, added legacy resources check, please check